include/signalrclient/hub_connection_builder.h

```diff
@@ -35,6 +35,10 @@ namespace signalr
 
         SIGNALRCLIENT_API hub_connection_builder& skip_negotiation(bool skip = true);
 
+#ifdef USE_MSGPACK
```

Comment:
Interesting that it's all build time (Yah, I suck at C++);

Reply:
Only because we don't want to include messgepack code when the user doesn't use it. And didn't want to have 2 different projects so it would be like the other clients.